spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -164,10 +165,11 @@ public TransactionInfo getTransactionInfo() {
         }
     }
 
-    private <T extends Annotation> T getAnnotation(Method method, Class<T> clazz) {
-        return method == null ? null : method.getAnnotation(clazz);
+    private <T extends Annotation> T getAnnotation(Method method, Class<?> clazz, Class<T> clz) {
```

Comment:
clazz variable to targetClass, clz variable to annotationClass ?

Reply:
ok